### PR TITLE
add an enumeration for Network types that could be referenced by NetworkOptions

### PR DESCRIPTION
**Pull request by rolfyone**
…orkOptions.

Addresses #1642

Signed-off-by: Paul Harris <paul.harris@consensys.net>

### CODE REVIEW

File: artemis/src/main/java/tech/pegasys/artemis/cli/options/NetworkOptions.java [hunk 1]
```diff
@@ -14,17 +14,18 @@
 package tech.pegasys.artemis.cli.options;
 
 import picocli.CommandLine.Option;
+import tech.pegasys.artemis.util.config.NetworkConfigurations;
 
 public class NetworkOptions {
 
   @Option(
```
Comment (mark-terry) [inline artemis/src/main/java/tech/pegasys/artemis/cli/options/NetworkOptions.java:21]: Should the default value be added here?

**Review by mark-terry — COMMENTED** [review]
One comment.

**Review by ajsutton — CHANGES_REQUESTED** [review]
Networks are not an enumeration.  `--network https://github.com/eth2-clients/eth2-testnets/raw/master/prysm/Sapphire(v0.9.4)/config.yaml` is totally valid.

**Comment by rolfyone** [comment]
> Networks are not an enumeration. `--network https://github.com/eth2-clients/eth2-testnets/raw/master/prysm/Sapphire(v0.9.4)/config.yaml` is totally valid.

being able to use --network in this way is good to know, and pretty much means the change I was planning is invalid. I'll close this PR, and potentially put in a test case to show this usage so we know about it in future.

**Pull request closed without merging**